[2023-02-28T01:03:40.160+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [queued]>
[2023-02-28T01:03:40.175+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [queued]>
[2023-02-28T01:03:40.175+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-28T01:03:40.175+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-28T01:03:40.175+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-28T01:03:40.195+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): make_directory> on 2023-02-27 00:00:00+00:00
[2023-02-28T01:03:40.204+0000] {standard_task_runner.py:55} INFO - Started process 706 to run task
[2023-02-28T01:03:40.209+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_date_in_file_txt', 'make_directory', 'scheduled__2023-02-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/print_date_file.py', '--cfg-path', '/tmp/tmppcvwqxqy']
[2023-02-28T01:03:40.210+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask make_directory
[2023-02-28T01:03:40.335+0000] {task_command.py:388} INFO - Running <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [running]> on host aca83939d072
[2023-02-28T01:03:40.473+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=save_date_in_file_txt
AIRFLOW_CTX_TASK_ID=make_directory
AIRFLOW_CTX_EXECUTION_DATE=2023-02-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-27T00:00:00+00:00
[2023-02-28T01:03:40.476+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-28T01:03:40.479+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'mkdir -p /home/pedrodesa/Área de Trabalho']
[2023-02-28T01:03:40.499+0000] {subprocess.py:86} INFO - Output:
[2023-02-28T01:03:40.506+0000] {subprocess.py:93} INFO - mkdir: cannot create directory ‘/home/pedrodesa’: Permission denied
[2023-02-28T01:03:40.507+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-02-28T01:03:40.536+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-02-28T01:03:40.546+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=save_date_in_file_txt, task_id=make_directory, execution_date=20230227T000000, start_date=20230228T010340, end_date=20230228T010340
[2023-02-28T01:03:40.577+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 10 for task make_directory (Bash command failed. The command returned a non-zero exit code 1.; 706)
[2023-02-28T01:03:40.637+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-28T01:03:40.713+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01T00:45:15.690+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [queued]>
[2023-03-01T00:45:15.739+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [queued]>
[2023-03-01T00:45:15.740+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:45:15.740+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-01T00:45:15.740+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-01T00:45:15.821+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): make_directory> on 2023-02-27 00:00:00+00:00
[2023-03-01T00:45:15.848+0000] {standard_task_runner.py:55} INFO - Started process 121 to run task
[2023-03-01T00:45:15.870+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_date_in_file_txt', 'make_directory', 'scheduled__2023-02-27T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/print_date_file.py', '--cfg-path', '/tmp/tmphxc9xt49']
[2023-03-01T00:45:15.871+0000] {standard_task_runner.py:83} INFO - Job 36: Subtask make_directory
[2023-03-01T00:45:16.149+0000] {task_command.py:388} INFO - Running <TaskInstance: save_date_in_file_txt.make_directory scheduled__2023-02-27T00:00:00+00:00 [running]> on host 60fce3bd3be2
[2023-03-01T00:45:16.459+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=save_date_in_file_txt
AIRFLOW_CTX_TASK_ID=make_directory
AIRFLOW_CTX_EXECUTION_DATE=2023-02-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-27T00:00:00+00:00
[2023-03-01T00:45:16.461+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-01T00:45:16.465+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'mkdir -p ./data']
[2023-03-01T00:45:16.529+0000] {subprocess.py:86} INFO - Output:
[2023-03-01T00:45:16.541+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-01T00:45:16.719+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=save_date_in_file_txt, task_id=make_directory, execution_date=20230227T000000, start_date=20230301T004515, end_date=20230301T004516
[2023-03-01T00:45:16.872+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-01T00:45:17.485+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
